tests/pytorch/test_pytorch_autolog.py

```diff
@@ -254,3 +252,19 @@ def test_pytorch_test_metrics_logged(pytorch_model_tests):
     data = run.data
     assert "test_loss" in data.metrics
     assert "test_acc" in data.metrics
+
+
+def test_get_optimizer_name():
+    adam = torch.optim.Adam(torch.nn.Linear(1, 1).parameters())
+    assert _get_optimizer_name(adam) == "Adam"
+
+
+@pytest.mark.skipif(
+    LooseVersion(pl.__version__) < LooseVersion("1.1.0"),
+    reason="`LightningOptimizer` doesn't exist in pytorch-lightning < 1.1.0",
+)
```

Comment:
Should we include `pytorch-lightning` in `ml-package-versions.yml` (I'd be happy to file a folllow-up PR)?